pom.xml

```diff
@@ -579,6 +579,14 @@
 				<version>3.0.0</version>
 			</dependency>
 
+			<!-- Because we use Spring-JCL we don't want Commons Logging. -->
```

Comment:
Fixed: `Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar to avoid ...`